src/main/java/ro/msg/learning/shop/utils/TestingUtils.java

```diff
@@ -0,0 +1,224 @@
+package ro.msg.learning.shop.utils;
+
+import ro.msg.learning.shop.entitites.*;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class TestingUtils {
+	private static final UUID customerId = UUID.fromString("631e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID location1Id = UUID.fromString("431e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID location2Id = UUID.fromString("431e4cdd-bb78-4769-a0c7-cb948a9f1232");
+	private static final UUID stock1Id = UUID.fromString("531e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID stock2Id = UUID.fromString("531e4cdd-bb78-4769-a0c7-cb948a9f1232");
+	private static final UUID supplier1Id = UUID.fromString("131e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID supplier2Id = UUID.fromString("131e4cdd-bb78-4769-a0c7-cb948a9f1232");
+	private static final UUID productCategory1Id = UUID.fromString("231e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID productCategory2Id = UUID.fromString("231e4cdd-bb78-4769-a0c7-cb948a9f1232");
+
+	private static final UUID product1Id = UUID.fromString("331e4cdd-bb78-4769-a0c7-cb948a9f1231");
+	private static final UUID product2Id = UUID.fromString("331e4cdd-bb78-4769-a0c7-cb948a9f1232");
+
+	public static Order createOrder(UUID productId1, UUID productId2) {
+		List<OrderDetail> orderDetails = new ArrayList<>();
+		OrderDetail orderDetail1 = new OrderDetail();
```

Comment:
Please use the constructors with parameters to create the objects, do not create an empty object and then use the setters. The code is unreadable

Reply:
Refactored all TestingUtils class.